include/bbp/sonata/config.h

```diff
@@ -645,6 +646,30 @@ class SONATA_API SimulationConfig
         bool representsPhysicalElectrode = false;
     };
 
+    struct EField {
+        /// Peak amplitude of the sinusoid in the x-direction, in V/m
+        double ex{};
+        /// Peak amplitude of the sinusoid in the y-direction, in V/m
+        double ey{};
+        /// Peak amplitude of the sinusoid in the z-direction, in V/m
+        double ez{};
+        /// Frequency of the sinusoid, in Hz. Must be non-negative. If not provided, assumed to be 0
+        double frequency{};
+    };
+
+    struct InputUniformEField: public InputBase {
+      public:
+        /// A list of dicts, where each dict defines one of the fields which are summed to produce
+        /// the total stimulus.
+        std::vector<EField> fields;
+        /// Duration during which the signal ramps up linearly from 0, in ms. Default is 0 ms.
+        double rampUpTime;
+        /// Duration during which the signal ramps down linearly from 0, in ms.
```

Comment:
ramp up and down... in what sense? you scale ex, ey, ez or the frequency? or both?